File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// WeatherDataApi3 SDK exists test

require_once __DIR__ . '/../weatherdataapi3_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = WeatherDataApi3SDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# WeatherDataApi3 SDK exists test

require "minitest/autorun"
require_relative "../WeatherDataApi3_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = WeatherDataApi3SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
